cmd/thanos/compact.go

```diff
@@ -21,6 +23,38 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+var (
+	compactions = compactionSet{
+		1 * time.Hour,
+		2 * time.Hour,
+		8 * time.Hour,
+		2 * 24 * time.Hour,
+		14 * 24 * time.Hour,
+	}
+)
+
+type compactionSet []time.Duration
+
+func (cs compactionSet) String() string {
+	result := make([]string, len(cs))
+	for i, c := range cs {
+		result[i] = fmt.Sprintf("%d=%dh", i, int(c.Hours()))
+	}
+	return strings.Join(result, ", ")
+}
+
+func (cs compactionSet) enabled(maxLevel int) ([]int64, error) {
```

Comment:
Why `enabled` verb? what we enable here? Maybe `levels` instead?

```suggestion
func (cs compactionSet) levels(maxLevel int) ([]int64, error) {
```

Reply:
assumed enabled levels, but levels instead should be ok